### ISSUE TITLE
gr.ligrec error: read_csv() got an unexpected keyword argument 'squeeze'

### ISSUE THREAD

**Issue by JuHey** [scverse/squidpy#688]
## Description

 trying to reproduce the gr.ligrec vignette as documented in: [https://workflows.omnipathdb.org/ligrec-squidpy.html](url). Installed squidpy, scanpy etc. via Mamba using conda-forge. When calling `res = sp.gr.ligrec(adata, "paul15_clusters", fdr_method=None, copy=True, interactions_params={"resources": "CellPhoneDB"}, threshold=0.1, seed=0, n_perms=10000, n_jobs=1)`, the process starts but then aborts with `TypeError: read_csv() got an unexpected keyword argument 'squeeze'`. My first hunch was that might be related to a pandas version issue as line 630 in the gr.ligrec code shows: `rec_scores_df = pd.read.csv(rec_scores_files, header=0, index_col=0, squeeze = True, sep = '\t')`. Is there a known incompatibility of squidpy with pandas version 2.0.1? How to troubleshoot this?

...

## Minimal reproducible example

```import numpy as np
import pandas as pd
import scanpy as sc
import squidpy as sp
from anndata import AnnData
adata = sc.datasets.paul15()
sc.pp.normalize_per_cell(adata)
adata.raw = adata.copy()
res = sp.gr.ligrec(adata, "paul15_clusters", fdr_method=None, copy=True, interactions_params={"resources": "CellPhoneDB"}, threshold=0.1, seed=0, n_perms=10000, n_jobs=1)

...
```

## Traceback

<TypeError                                 Traceback (most recent call last)
Cell In[53], line 1
----> 1 res = sp.gr.ligrec(adata, "paul15_clusters", fdr_method=None, copy=True, interactions_params={"resources": "CellPhoneDB"}, threshold=0.1, seed=0, n_perms=10000, n_jobs=1)

File ~/mambaforge/envs/squidpy/lib/python3.9/site-packages/squidpy/gr/_ligrec.py:662, in ligrec(adata, cluster_key, interactions, complex_policy, threshold, corr_method, corr_axis, use_raw, copy, key_added, gene_symbols, **kwargs)
    645 """
    646 %(PT_test.full_desc)s
    647 
   (...)
    658 %(ligrec_test_returns)s
    659 """  # noqa: D400
    660 with _genesymbols(adata, key=gene_symbols, use_raw=use_raw, make_unique=False):
    661     return (  # type: ignore[no-any-return]
--> 662         PermutationTest(adata, use_raw=use_raw)
    663         .prepare(interactions, complex_policy=complex_policy, **kwargs)
    664         .test(
    665             cluster_key=cluster_key,
    666             threshold=threshold,
    667             corr_method=corr_method,
    668             corr_axis=corr_axis,
    669             copy=copy,
    670             key_added=key_added,
    671             **kwargs,
    672         )
    673     )

File ~/mambaforge/envs/squidpy/lib/python3.9/site-packages/squidpy/gr/_ligrec.py:604, in PermutationTest.prepare(self, interactions, complex_policy, interactions_params, transmitter_params, receiver_params, **_)
    601 from omnipath.interactions import import_intercell_network
    603 start = logg.info("Fetching interactions from `omnipath`")
--> 604 interactions = import_intercell_network(
    605     interactions_params=interactions_params,
    606     transmitter_params=transmitter_params,
    607     receiver_params=receiver_params,
    608 )
    609 if TYPE_CHECKING:
    610     assert isinstance(interactions, pd.DataFrame)

File ~/mambaforge/envs/squidpy/lib/python3.9/site-packages/omnipath/_core/requests/interactions/_utils.py:102, in import_intercell_network(include, interactions_params, transmitter_params, receiver_params)
     99 receiver_params.setdefault("causality", "rec")
    100 receiver_params.setdefault("scope", "generic")
--> 102 interactions = AllInteractions.get(include=include, **interactions_params)
    103 if interactions.empty:
    104     raise ValueError(_ERROR_EMPTY_FMT.format(obj="interactions"))

File ~/mambaforge/envs/squidpy/lib/python3.9/site-packages/omnipath/_core/requests/_utils.py:104, in _inject_api_method.<locals>.wrapper(wrapped, _instance, args, kwargs)
    102 @wrapt.decorator(adapter=wrapt.adapter_factory(argspec_factory))
    103 def wrapper(wrapped, _instance, args, kwargs):
--> 104     return wrapped(*args, **kwargs)

File ~/mambaforge/envs/squidpy/lib/python3.9/site-packages/omnipath/_core/requests/interactions/_interactions.py:377, in AllInteractions.get(cls, include, exclude, **kwargs)
    352 @classmethod
    353 @d.dedent
    354 def get(
   (...)
    358     **kwargs,
    359 ) -> pd.DataFrame:
    360     """
    361     %(get.full_desc)s
    362 
   (...)
    375     %(get.returns)s
    376     """
--> 377     return cls(include, exclude=exclude)._get(**kwargs)

File ~/mambaforge/envs/squidpy/lib/python3.9/site-packages/omnipath/_core/requests/_request.py:107, in OmnipathRequestABC._get(self, **kwargs)
    104 kwargs = self._validate_params(kwargs)
    105 kwargs = self._finalize_params(kwargs)
--> 107 res = self._downloader.maybe_download(
    108     self._query_type.endpoint, params=kwargs, callback=callback, is_final=False
    109 )
    111 if self._downloader._options.convert_dtypes:
    112     res = self._convert_dtypes(res)

File ~/mambaforge/envs/squidpy/lib/python3.9/site-packages/omnipath/_core/downloader/_downloader.py:128, in Downloader.maybe_download(self, url, callback, params, cache, is_final, **_)
    126     res = self._options.cache[key]
    127 else:
--> 128     res = callback(self._download(req))
    129     if cache:
    130         logging.debug(f"Caching result to `{self._options.cache}[{key!r}]`")

File ~/mambaforge/envs/squidpy/lib/python3.9/site-packages/omnipath/_core/requests/_request.py:39, in _error_handler.<locals>.wrapper(cls, *args, **kwargs)
     37 @wraps(callback)
     38 def wrapper(cls, *args, **kwargs) -> pd.DataFrame:
---> 39     res: pd.DataFrame = callback(*args, **kwargs)
     40     if len(res.columns) == 1 and res.columns == ["Something is not entirely good:"]:
     41         raise RuntimeError(" ".join(res.iloc[:, 0]))

TypeError: read_csv() got an unexpected keyword argument 'squeeze'>

```pytb
...
```

</details>

## Version

1.2.3
...

**Comment by michalk8** [comment]
This was fixed in https://github.com/saezlab/omnipath/pull/40, updating `omnipath` from `master` should make this work. Will try to make a new release in a couple of weeks.

**Comment by michalk8** [comment]
Should be fixed in `omnipath==1.0.7`!
